doc/widgets/classify/classificationtreeviewer.md

```diff
@@ -0,0 +1,71 @@
+Classification Tree Viewer
+==========================
+
+![image](icons/classification-tree-viewer.png)
+
+Visualization of classification tree.
+
+Signals
+-------
+
+**Inputs**:
+
+- **Classification Tree**
+
+  Classification tree.
+
+**Outputs**:
+
+- **Data**
+
+  Data from the selected tree node.
+
+Description
+-----------
+
+This is a versatile widget with 2-D visualization of a [classification tree](https://en.wikipedia.org/wiki/Decision_tree_learning).
+The user can select a node, instructing the widget to output the data
+associated with the node thus enabling explorative data analysis.
+
+![Classification Tree Graph](images/ClassificationTreeViewer-stamped.png)
+
+1. Information on the input.
+
+2. Set the zoom and define the tree width. Nodes display
+  information bubble when hovering over them.
+
+3. Edges between nodes in the tree graph are drawn based on the selected
+  edge width.
+    - All the edges will be of equal width if *Fixed* is chosen.
+    - When *Relative to root* is selected, the width of the edge will correspond to
+    the proportion of instances in the corresponding node with respect to
+    all the instances in the training data. Under this selection the edge will get thinner and
+  thinner when traversing toward the bottom of the tree.
```

Comment:
Agree, but editor requires indenture for sub-lists. Otherwise it continues the count as main list (1., 2., 3....).